src/Couchbase.Lite.Shared/API/Sync/Replicator.cs

```diff
@@ -398,16 +393,6 @@ private void WaitPendingConflictTasks(C4ReplicatorStatus status)
             }
         }
 
-        private void ClearRepl()
-        {
-            DispatchQueue.DispatchSync(() =>
-            {
-                Native.c4repl_free(_repl);
```

Comment:
Where is this freed now?

Reply:
Native.c4repl_free(_repl); is now called in Dispose/Finalizing.

Reply:
Oh ok, that was already there!